src/views/Search.vue

```diff
@@ -81,16 +81,23 @@ export default defineComponent({
   },
   data (){
     return {
-      queryString: ''
+      queryString: (this as any).searchQuery
```

Comment:
```suggestion
      queryString: ''
```
The rest looks good to me.